.changeset/lemon-clouds-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sl-design-system/website": patch
```

Comment:
We are adding changeset only for npm packages: https://github.com/orgs/sl-design-system/packages?repo_name=components
So the changeset is not necessary for the website changes.
Can you please remove this file? :)